[tesseract] Build error

Package: tesseract:arm64-linux -> 5.3.1

**Host Environment**

- Host: arm64-linux
- Compiler: GNU 8.4.0
-    vcpkg-tool version: 2999-12-31-unknownhash
    vcpkg-scripts version: f6a5d4e8e 2023-06-20 (7 days ago)

**To Reproduce**

`vcpkg install boost blas openblas[threads] catch2 cereal ceres cryptopp eigen3 g2o gflags glog glm glew glfw3 hidapi imgui[glfw-binding,opengl3-binding] implot qhull sqlite3 websocketpp yaml-cpp openssl opencv opencv4[contrib,cuda,dnn,cudnn,dnn-cuda,eigen,jasper,jpeg,png,quirc,tbb,tiff,webp]`

**Failure logs**

```
-- Using cached tesseract-ocr-tesseract-5.3.1.tar.gz.
-- Cleaning sources at /media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source /media/tripleye/vcpkg-tripleye-fork/downloads/tesseract-ocr-tesseract-5.3.1.tar.gz
-- Applying patch fix-depend-libarchive.patch
-- Applying patch fix_static_link_icu.patch
-- Using source at /media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean
-- Configuring arm64-linux
-- Building arm64-linux-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: /usr/local/bin/cmake --build . --config Debug --target install -- -v -j7
    Working Directory: /media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/arm64-linux-dbg
    See logs for more information:
      /media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/install-arm64-linux-dbg-out.log

Call Stack (most recent call first):
  installed/arm64-linux/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  installed/arm64-linux/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/tesseract/portfile.cmake:35 (vcpkg_cmake_install)
  scripts/ports.cmake:147 (include)



```
<details><summary>/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/install-arm64-linux-dbg-out.log</summary>

```
[1/219] /usr/bin/c++ -DCMAKE_BUILD -DHAVE_CONFIG_H -DLIBXML_STATIC -D_SILENCE_STDEXT_HASH_DEPRECATION_WARNINGS=1 -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/include -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/arch -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccmain -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccstruct -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccutil -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/classify -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/cutil -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/dict -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/lstm -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/opencl -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/textord -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/viewer -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/wordrec -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/training -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src -I/media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include/leptonica -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/arm64-linux-dbg -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/arm64-linux-dbg/include -isystem /media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include -isystem /media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include/libxml2 -fPIC -g -Wall -DDEBUG -pedantic -Og -std=gnu++2a -pthread -MD -MT CMakeFiles/libtesseract.dir/src/ccmain/adaptions.cpp.o -MF CMakeFiles/libtesseract.dir/src/ccmain/adaptions.cpp.o.d -o CMakeFiles/libtesseract.dir/src/ccmain/adaptions.cpp.o -c /media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccmain/adaptions.cpp
[2/219] /usr/bin/c++ -DCMAKE_BUILD -DHAVE_CONFIG_H -DLIBXML_STATIC -D_SILENCE_STDEXT_HASH_DEPRECATION_WARNINGS=1 -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/include -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/arch -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccmain -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccstruct -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccutil -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/classify -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/cutil -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/dict -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/lstm -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/opencl -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/textord -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/viewer -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/wordrec -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/training -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src -I/media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include/leptonica -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/arm64-linux-dbg -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/arm64-linux-dbg/include -isystem /media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include -isystem /media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include/libxml2 -fPIC -g -Wall -DDEBUG -pedantic -Og -std=gnu++2a -pthread -MD -MT CMakeFiles/libtesseract.dir/src/ccmain/fixxht.cpp.o -MF CMakeFiles/libtesseract.dir/src/ccmain/fixxht.cpp.o.d -o CMakeFiles/libtesseract.dir/src/ccmain/fixxht.cpp.o -c /media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccmain/fixxht.cpp
...
Skipped 194 lines
...

[197/219] /usr/bin/c++ -DCMAKE_BUILD -DHAVE_CONFIG_H -DLIBXML_STATIC -D_SILENCE_STDEXT_HASH_DEPRECATION_WARNINGS=1 -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/include -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/arch -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccmain -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccstruct -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccutil -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/classify -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/cutil -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/dict -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/lstm -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/opencl -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/textord -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/viewer -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/wordrec -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/training -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src -I/media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include/leptonica -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/arm64-linux-dbg -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/arm64-linux-dbg/include -isystem /media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include -isystem /media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include/libxml2 -fPIC -g -Wall -DDEBUG -pedantic -Og -std=gnu++2a -pthread -MD -MT CMakeFiles/libtesseract.dir/src/wordrec/pieces.cpp.o -MF CMakeFiles/libtesseract.dir/src/wordrec/pieces.cpp.o.d -o CMakeFiles/libtesseract.dir/src/wordrec/pieces.cpp.o -c /media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/wordrec/pieces.cpp
[198/219] /usr/bin/c++ -DCMAKE_BUILD -DHAVE_CONFIG_H -DLIBXML_STATIC -D_SILENCE_STDEXT_HASH_DEPRECATION_WARNINGS=1 -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/include -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/arch -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccmain -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccstruct -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccutil -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/classify -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/cutil -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/dict -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/lstm -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/opencl -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/textord -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/viewer -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/wordrec -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/training -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src -I/media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include/leptonica -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/arm64-linux-dbg -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/arm64-linux-dbg/include -isystem /media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include -isystem /media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include/libxml2 -fPIC -g -Wall -DDEBUG -pedantic -Og -std=gnu++2a -pthread -MD -MT CMakeFiles/libtesseract.dir/src/wordrec/render.cpp.o -MF CMakeFiles/libtesseract.dir/src/wordrec/render.cpp.o.d -o CMakeFiles/libtesseract.dir/src/wordrec/render.cpp.o -c /media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/wordrec/render.cpp
[199/219] /usr/bin/c++ -DCMAKE_BUILD -DHAVE_CONFIG_H -DLIBXML_STATIC -D_SILENCE_STDEXT_HASH_DEPRECATION_WARNINGS=1 -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/include -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/arch -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccmain -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccstruct -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccutil -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/classify -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/cutil -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/dict -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/lstm -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/opencl -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/textord -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/viewer -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/wordrec -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/training -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src -I/media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include/leptonica -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/arm64-linux-dbg -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/arm64-linux-dbg/include -isystem /media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include -isystem /media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include/libxml2 -fPIC -g -Wall -DDEBUG -pedantic -Og -std=gnu++2a -pthread -MD -MT CMakeFiles/libtesseract.dir/src/api/capi.cpp.o -MF CMakeFiles/libtesseract.dir/src/api/capi.cpp.o.d -o CMakeFiles/libtesseract.dir/src/api/capi.cpp.o -c /media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/api/capi.cpp
[200/219] /usr/bin/c++ -DCMAKE_BUILD -DHAVE_CONFIG_H -DLIBXML_STATIC -D_SILENCE_STDEXT_HASH_DEPRECATION_WARNINGS=1 -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/include -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/arch -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccmain -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccstruct -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccutil -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/classify -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/cutil -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/dict -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/lstm -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/opencl -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/textord -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/viewer -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/wordrec -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/training -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src -I/media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include/leptonica -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/arm64-linux-dbg -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/arm64-linux-dbg/include -isystem /media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include -isystem /media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include/libxml2 -fPIC -g -Wall -DDEBUG -pedantic -Og -std=gnu++2a -pthread -MD -MT CMakeFiles/libtesseract.dir/src/api/renderer.cpp.o -MF CMakeFiles/libtesseract.dir/src/api/renderer.cpp.o.d -o CMakeFiles/libtesseract.dir/src/api/renderer.cpp.o -c /media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/api/renderer.cpp
[201/219] /usr/bin/c++ -DCMAKE_BUILD -DHAVE_CONFIG_H -DLIBXML_STATIC -D_SILENCE_STDEXT_HASH_DEPRECATION_WARNINGS=1 -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/include -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/arch -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccmain -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccstruct -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccutil -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/classify -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/cutil -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/dict -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/lstm -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/opencl -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/textord -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/viewer -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/wordrec -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/training -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src -I/media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include/leptonica -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/arm64-linux-dbg -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/arm64-linux-dbg/include -isystem /media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include -isystem /media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include/libxml2 -fPIC -g -Wall -DDEBUG -pedantic -Og -std=gnu++2a -pthread -MD -MT CMakeFiles/libtesseract.dir/src/wordrec/tface.cpp.o -MF CMakeFiles/libtesseract.dir/src/wordrec/tface.cpp.o.d -o CMakeFiles/libtesseract.dir/src/wordrec/tface.cpp.o -c /media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/wordrec/tface.cpp
[202/219] /usr/bin/c++ -DCMAKE_BUILD -DHAVE_CONFIG_H -DLIBXML_STATIC -D_SILENCE_STDEXT_HASH_DEPRECATION_WARNINGS=1 -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/include -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/arch -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccmain -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccstruct -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccutil -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/classify -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/cutil -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/dict -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/lstm -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/opencl -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/textord -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/viewer -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/wordrec -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/training -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src -I/media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include/leptonica -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/arm64-linux-dbg -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/arm64-linux-dbg/include -isystem /media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include -isystem /media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include/libxml2 -fPIC -g -Wall -DDEBUG -pedantic -Og -std=gnu++2a -pthread -MD -MT CMakeFiles/libtesseract.dir/src/wordrec/wordclass.cpp.o -MF CMakeFiles/libtesseract.dir/src/wordrec/wordclass.cpp.o.d -o CMakeFiles/libtesseract.dir/src/wordrec/wordclass.cpp.o -c /media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/wordrec/wordclass.cpp
[203/219] /usr/bin/c++ -DCMAKE_BUILD -DHAVE_CONFIG_H -DLIBXML_STATIC -D_SILENCE_STDEXT_HASH_DEPRECATION_WARNINGS=1 -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/include -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/arch -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccmain -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccstruct -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccutil -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/classify -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/cutil -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/dict -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/lstm -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/opencl -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/textord -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/viewer -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/wordrec -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/training -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src -I/media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include/leptonica -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/arm64-linux-dbg -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/arm64-linux-dbg/include -isystem /media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include -isystem /media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include/libxml2 -fPIC -g -Wall -DDEBUG -pedantic -Og -std=gnu++2a -pthread -MD -MT CMakeFiles/libtesseract.dir/src/wordrec/segsearch.cpp.o -MF CMakeFiles/libtesseract.dir/src/wordrec/segsearch.cpp.o.d -o CMakeFiles/libtesseract.dir/src/wordrec/segsearch.cpp.o -c /media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/wordrec/segsearch.cpp
[204/219] /usr/bin/c++ -DCMAKE_BUILD -DHAVE_CONFIG_H -DLIBXML_STATIC -D_SILENCE_STDEXT_HASH_DEPRECATION_WARNINGS=1 -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/include -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/arch -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccmain -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccstruct -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccutil -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/classify -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/cutil -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/dict -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/lstm -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/opencl -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/textord -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/viewer -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/wordrec -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/training -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src -I/media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include/leptonica -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/arm64-linux-dbg -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/arm64-linux-dbg/include -isystem /media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include -isystem /media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include/libxml2 -fPIC -g -Wall -DDEBUG -pedantic -Og -std=gnu++2a -pthread -MD -MT CMakeFiles/libtesseract.dir/src/wordrec/wordrec.cpp.o -MF CMakeFiles/libtesseract.dir/src/wordrec/wordrec.cpp.o.d -o CMakeFiles/libtesseract.dir/src/wordrec/wordrec.cpp.o -c /media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/wordrec/wordrec.cpp
[205/219] /usr/bin/c++ -DCMAKE_BUILD -DHAVE_CONFIG_H -DLIBXML_STATIC -D_SILENCE_STDEXT_HASH_DEPRECATION_WARNINGS=1 -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/include -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/arch -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccmain -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccstruct -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccutil -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/classify -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/cutil -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/dict -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/lstm -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/opencl -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/textord -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/viewer -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/wordrec -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/training -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src -I/media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include/leptonica -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/arm64-linux-dbg -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/arm64-linux-dbg/include -isystem /media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include -isystem /media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include/libxml2 -fPIC -g -Wall -DDEBUG -pedantic -Og -std=gnu++2a -pthread  -O3 -ffast-math -MD -MT CMakeFiles/libtesseract.dir/src/arch/dotproduct.cpp.o -MF CMakeFiles/libtesseract.dir/src/arch/dotproduct.cpp.o.d -o CMakeFiles/libtesseract.dir/src/arch/dotproduct.cpp.o -c /media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/arch/dotproduct.cpp
[206/219] /usr/bin/c++ -DCMAKE_BUILD -DHAVE_CONFIG_H -DLIBXML_STATIC -D_SILENCE_STDEXT_HASH_DEPRECATION_WARNINGS=1 -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/include -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/arch -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccmain -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccstruct -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccutil -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/classify -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/cutil -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/dict -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/lstm -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/opencl -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/textord -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/viewer -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/wordrec -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/training -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src -I/media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include/leptonica -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/arm64-linux-dbg -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/arm64-linux-dbg/include -isystem /media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include -isystem /media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include/libxml2 -fPIC -g -Wall -DDEBUG -pedantic -Og -std=gnu++2a -pthread -MD -MT CMakeFiles/libtesseract.dir/src/api/altorenderer.cpp.o -MF CMakeFiles/libtesseract.dir/src/api/altorenderer.cpp.o.d -o CMakeFiles/libtesseract.dir/src/api/altorenderer.cpp.o -c /media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/api/altorenderer.cpp
[207/219] /usr/bin/c++ -DCMAKE_BUILD -DHAVE_CONFIG_H -DLIBXML_STATIC -D_SILENCE_STDEXT_HASH_DEPRECATION_WARNINGS=1 -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/include -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/arch -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccmain -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccstruct -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccutil -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/classify -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/cutil -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/dict -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/lstm -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/opencl -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/textord -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/viewer -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/wordrec -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/training -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src -I/media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include/leptonica -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/arm64-linux-dbg -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/arm64-linux-dbg/include -isystem /media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include -isystem /media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include/libxml2 -fPIC -g -Wall -DDEBUG -pedantic -Og -std=gnu++2a -pthread -MD -MT CMakeFiles/libtesseract.dir/src/arch/simddetect.cpp.o -MF CMakeFiles/libtesseract.dir/src/arch/simddetect.cpp.o.d -o CMakeFiles/libtesseract.dir/src/arch/simddetect.cpp.o -c /media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/arch/simddetect.cpp
[208/219] /usr/bin/c++ -DCMAKE_BUILD -DHAVE_CONFIG_H -DLIBXML_STATIC -D_SILENCE_STDEXT_HASH_DEPRECATION_WARNINGS=1 -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/include -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/arch -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccmain -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccstruct -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccutil -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/classify -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/cutil -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/dict -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/lstm -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/opencl -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/textord -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/viewer -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/wordrec -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/training -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src -I/media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include/leptonica -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/arm64-linux-dbg -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/arm64-linux-dbg/include -isystem /media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include -isystem /media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include/libxml2 -fPIC -g -Wall -DDEBUG -pedantic -Og -std=gnu++2a -pthread -mfpu=neon -MD -MT CMakeFiles/libtesseract.dir/src/arch/dotproductneon.cpp.o -MF CMakeFiles/libtesseract.dir/src/arch/dotproductneon.cpp.o.d -o CMakeFiles/libtesseract.dir/src/arch/dotproductneon.cpp.o -c /media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/arch/dotproductneon.cpp
FAILED: CMakeFiles/libtesseract.dir/src/arch/dotproductneon.cpp.o
/usr/bin/c++ -DCMAKE_BUILD -DHAVE_CONFIG_H -DLIBXML_STATIC -D_SILENCE_STDEXT_HASH_DEPRECATION_WARNINGS=1 -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/include -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/arch -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccmain -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccstruct -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccutil -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/classify -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/cutil -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/dict -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/lstm -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/opencl -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/textord -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/viewer -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/wordrec -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/training -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src -I/media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include/leptonica -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/arm64-linux-dbg -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/arm64-linux-dbg/include -isystem /media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include -isystem /media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include/libxml2 -fPIC -g -Wall -DDEBUG -pedantic -Og -std=gnu++2a -pthread -mfpu=neon -MD -MT CMakeFiles/libtesseract.dir/src/arch/dotproductneon.cpp.o -MF CMakeFiles/libtesseract.dir/src/arch/dotproductneon.cpp.o.d -o CMakeFiles/libtesseract.dir/src/arch/dotproductneon.cpp.o -c /media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/arch/dotproductneon.cpp
c++: error: unrecognized command line option ‘-mfpu=neon’
[209/219] /usr/bin/c++ -DCMAKE_BUILD -DHAVE_CONFIG_H -DLIBXML_STATIC -D_SILENCE_STDEXT_HASH_DEPRECATION_WARNINGS=1 -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/include -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/arch -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccmain -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccstruct -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccutil -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/classify -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/cutil -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/dict -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/lstm -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/opencl -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/textord -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/viewer -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/wordrec -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/training -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src -I/media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include/leptonica -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/arm64-linux-dbg -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/arm64-linux-dbg/include -isystem /media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include -isystem /media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include/libxml2 -fPIC -g -Wall -DDEBUG -pedantic -Og -std=gnu++2a -pthread -MD -MT CMakeFiles/libtesseract.dir/src/api/lstmboxrenderer.cpp.o -MF CMakeFiles/libtesseract.dir/src/api/lstmboxrenderer.cpp.o.d -o CMakeFiles/libtesseract.dir/src/api/lstmboxrenderer.cpp.o -c /media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/api/lstmboxrenderer.cpp
[210/219] /usr/bin/c++ -DCMAKE_BUILD -DHAVE_CONFIG_H -DLIBXML_STATIC -D_SILENCE_STDEXT_HASH_DEPRECATION_WARNINGS=1 -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/include -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/arch -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccmain -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccstruct -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccutil -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/classify -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/cutil -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/dict -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/lstm -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/opencl -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/textord -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/viewer -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/wordrec -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/training -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src -I/media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include/leptonica -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/arm64-linux-dbg -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/arm64-linux-dbg/include -isystem /media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include -isystem /media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include/libxml2 -fPIC -g -Wall -DDEBUG -pedantic -Og -std=gnu++2a -pthread -MD -MT CMakeFiles/libtesseract.dir/src/arch/intsimdmatrix.cpp.o -MF CMakeFiles/libtesseract.dir/src/arch/intsimdmatrix.cpp.o.d -o CMakeFiles/libtesseract.dir/src/arch/intsimdmatrix.cpp.o -c /media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/arch/intsimdmatrix.cpp
[211/219] /usr/bin/c++ -DCMAKE_BUILD -DHAVE_CONFIG_H -DLIBXML_STATIC -D_SILENCE_STDEXT_HASH_DEPRECATION_WARNINGS=1 -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/include -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/arch -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccmain -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccstruct -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccutil -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/classify -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/cutil -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/dict -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/lstm -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/opencl -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/textord -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/viewer -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/wordrec -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/training -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src -I/media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include/leptonica -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/arm64-linux-dbg -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/arm64-linux-dbg/include -isystem /media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include -isystem /media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include/libxml2 -fPIC -g -Wall -DDEBUG -pedantic -Og -std=gnu++2a -pthread -MD -MT CMakeFiles/libtesseract.dir/src/api/hocrrenderer.cpp.o -MF CMakeFiles/libtesseract.dir/src/api/hocrrenderer.cpp.o.d -o CMakeFiles/libtesseract.dir/src/api/hocrrenderer.cpp.o -c /media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/api/hocrrenderer.cpp
[212/219] /usr/bin/c++ -DCMAKE_BUILD -DHAVE_CONFIG_H -DLIBXML_STATIC -D_SILENCE_STDEXT_HASH_DEPRECATION_WARNINGS=1 -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/include -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/arch -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccmain -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccstruct -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccutil -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/classify -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/cutil -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/dict -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/lstm -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/opencl -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/textord -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/viewer -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/wordrec -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/training -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src -I/media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include/leptonica -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/arm64-linux-dbg -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/arm64-linux-dbg/include -isystem /media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include -isystem /media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include/libxml2 -fPIC -g -Wall -DDEBUG -pedantic -Og -std=gnu++2a -pthread -MD -MT CMakeFiles/libtesseract.dir/src/api/wordstrboxrenderer.cpp.o -MF CMakeFiles/libtesseract.dir/src/api/wordstrboxrenderer.cpp.o.d -o CMakeFiles/libtesseract.dir/src/api/wordstrboxrenderer.cpp.o -c /media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/api/wordstrboxrenderer.cpp
[213/219] /usr/bin/c++ -DCMAKE_BUILD -DHAVE_CONFIG_H -DLIBXML_STATIC -D_SILENCE_STDEXT_HASH_DEPRECATION_WARNINGS=1 -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/include -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/arch -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccmain -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccstruct -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccutil -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/classify -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/cutil -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/dict -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/lstm -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/opencl -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/textord -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/viewer -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/wordrec -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/training -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src -I/media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include/leptonica -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/arm64-linux-dbg -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/arm64-linux-dbg/include -isystem /media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include -isystem /media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include/libxml2 -fPIC -g -Wall -DDEBUG -pedantic -Og -std=gnu++2a -pthread -MD -MT CMakeFiles/libtesseract.dir/src/api/pdfrenderer.cpp.o -MF CMakeFiles/libtesseract.dir/src/api/pdfrenderer.cpp.o.d -o CMakeFiles/libtesseract.dir/src/api/pdfrenderer.cpp.o -c /media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/api/pdfrenderer.cpp
[214/219] /usr/bin/c++ -DCMAKE_BUILD -DHAVE_CONFIG_H -DLIBXML_STATIC -D_SILENCE_STDEXT_HASH_DEPRECATION_WARNINGS=1 -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/include -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/arch -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccmain -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccstruct -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/ccutil -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/classify -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/cutil -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/dict -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/lstm -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/opencl -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/textord -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/viewer -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/wordrec -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/training -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src -I/media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include/leptonica -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/arm64-linux-dbg -I/media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/arm64-linux-dbg/include -isystem /media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include -isystem /media/tripleye/vcpkg-tripleye-fork/installed/arm64-linux/include/libxml2 -fPIC -g -Wall -DDEBUG -pedantic -Og -std=gnu++2a -pthread -MD -MT CMakeFiles/libtesseract.dir/src/api/baseapi.cpp.o -MF CMakeFiles/libtesseract.dir/src/api/baseapi.cpp.o.d -o CMakeFiles/libtesseract.dir/src/api/baseapi.cpp.o -c /media/tripleye/vcpkg-tripleye-fork/buildtrees/tesseract/src/5.3.1-a5892682bb.clean/src/api/baseapi.cpp
ninja: build stopped: subcommand failed.
```
</details>

**Additional context**
According to [stackoverflow](https://stackoverflow.com/questions/29851128/gcc-arm64-aarch64-unrecognized-command-line-option-mfpu-neon) the -mfpu=neon is not required and without it, the compilation succeeds.
Don't know if this can be fixed on vcpkgs side or has to be done in tesseract.

--- a/ports/tesseract/fix-aarch64-mfpu-not-available.patch
+++ b/ports/tesseract/fix-aarch64-mfpu-not-available.patch
@@ -1,0 +1,25 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index db4c39d5..276c65f3 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -257,7 +257,11 @@ elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "arm.*")
+   set(HAVE_AVX512F FALSE)
+   set(HAVE_FMA FALSE)
+   set(HAVE_SSE4_1 FALSE)
++
+   check_cxx_compiler_flag("-mfpu=neon" HAVE_NEON)
++  if(HAVE_NEON)
++    set(NEON_COMPILE_FLAGS "-mfpu=neon")
++  endif(HAVE_NEON)
+ 
+ else()
+ 
+@@ -271,7 +275,6 @@ else()
+ endif(CMAKE_SYSTEM_PROCESSOR MATCHES "x86|x86_64|AMD64|amd64|i386|i686")
+ 
+ if(HAVE_NEON)
+-  set(NEON_COMPILE_FLAGS "-mfpu=neon")
+   message(STATUS "LTO build is not supported on arm/RBPi.")
+   set(ENABLE_LTO FALSE)  # enable LTO cause fatal error on arm/RBPi
+ endif()
+ 

--- a/ports/tesseract/portfile.cmake
+++ b/ports/tesseract/portfile.cmake
@@ -10,6 +10,7 @@ vcpkg_from_github(
     PATCHES
         ${tesseract_patch}
         fix_static_link_icu.patch
+        fix-aarch64-mfpu-not-available.patch
 )
 
 vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS

--- a/ports/tesseract/vcpkg.json
+++ b/ports/tesseract/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "tesseract",
   "version": "5.3.1",
+  "port-version": 1,
   "description": "An OCR Engine that was developed at HP Labs between 1985 and 1995... and now at Google.",
   "homepage": "https://github.com/tesseract-ocr/tesseract",
   "license": "Apache-2.0",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -7994,7 +7994,7 @@
     },
     "tesseract": {
       "baseline": "5.3.1",
-      "port-version": 0
+      "port-version": 1
     },
     "tfhe": {
       "baseline": "1.0.1",

--- a/versions/t-/tesseract.json
+++ b/versions/t-/tesseract.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "d0b39327eee123cac1611afe9d88303822fedbdf",
+      "version": "5.3.1",
+      "port-version": 1
+    },
+    {
       "git-tree": "aeb035e40f678090353d237cc6de1089fb58d89c",
       "version": "5.3.1",
       "port-version": 0

